src/lib/i18n/texts/de.json

```diff
@@ -102,6 +107,53 @@
     "skinLocked": "Skin gesperrt",
     "requiredLevel": "Benötigtes Level",
     "weaponStrength": "Waffenstärke",
-    "currentlyEquipped": "Aktuell ausgerüstet"
+    "defense": "Verteidigung",
+    "currentlyEquipped": "Aktuell ausgerüstet",
+    "accountBound": "Accountgebunden",
+    "soulBound": "Seelengebunden"
+  },
+  "itemAttributes": {
+    "power": "Kraft",
+    "precision": "Präzision",
+    "toughness": "Zähigkeit",
+    "vitality": "Vitalität",
+    "boonDuration": "Segensdauer",
+    "conditionDamage": "Zustandsschaden",
+    "conditionDuration": "Zustandsdauer",
+    "ferocity": "Wildheit",
+    "healingPower": "Heilkraft",
+    "armor": "Rüstung",
+    "critChance": "Kritische Trefferchance",
+    "critDamage": "Kritischer Schaden",
+    "health": "Lebenspunkte",
+    "expertise": "Fachkenntnis",
+    "concentration": "Konzentration",
+    "agonyResistance": "Qual-Widerstand",
+    "magic": "Magisches Gespür",
+    "elementalist": "Klassenspezifisches Attribut",
```

Comment:
no but i can't find the correct special class attributes in german ._. i will update this later but for now i think it's enough :/